src/inputs/dnstap/DnstapInputStream.h

```diff
@@ -11,19 +11,15 @@
 #include <sigslot/signal.hpp>
 #include <spdlog/spdlog.h>
 #include <unordered_map>
-#include <uv.h>
-
-namespace uvw {
-class Loop;
-class AsyncHandle;
-class PipeHandle;
-class TCPHandle;
-}
+#include <uvw.hpp>
```

Comment:
curious why this change?

Reply:
We are using uvw which is a c++ wrapper over libuv. By including the wrapper header intead of libuv we dont need to declare the classes that exists on uvw (TCPHandle, etc).

Reply:
right I did this originally as a compile time optimization though - uvw includes a lot of things we don't need in the interface definition here.

Reply:
Oh sorry. I just remove my changes in this part.